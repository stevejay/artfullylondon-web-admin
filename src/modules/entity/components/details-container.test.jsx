import React from 'react'
import { shallow } from 'enzyme'

import EntityDetailsContainer from './details-container'

it('should render a default container correctly', () => {
  const wrapper = shallow(
    <EntityDetailsContainer>
      <div id='child' />
    </EntityDetailsContainer>
  )

  expect(wrapper).toMatchSnapshot()
})

it('should render a narrow container correctly', () => {
  const wrapper = shallow(
    <EntityDetailsContainer type='narrow'>
      <div id='child' />
    </EntityDetailsContainer>
  )

  expect(wrapper).toMatchSnapshot()
})
