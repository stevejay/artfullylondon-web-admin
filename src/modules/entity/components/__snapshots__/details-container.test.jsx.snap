// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a default container correctly 1`] = `
<div
  className="src-modules-entity-components-details-container__container-default src-modules-entity-components-details-container__container"
>
  <div
    id="child"
  />
</div>
`;

exports[`should render a narrow container correctly 1`] = `
<div
  className="src-modules-entity-components-details-container__container-narrow src-modules-entity-components-details-container__container"
>
  <div
    id="child"
  />
</div>
`;
